test/contracts/OVM/accounts/OVM_ECDSAContractAccount.spec.ts

```diff
@@ -10,6 +10,8 @@ import { NON_ZERO_ADDRESS } from '../../../helpers/constants'
 import {
   serializeNativeTransaction,
   signNativeTransaction,
+  // `DEFAULT_EIP155_TX` comes with `chainId: 420` encoded as a key-value pair,
```

Comment:
I'm not clear about the rationale for this?
It feels a bit odd to me to be annotating this one module for what it imports, but not all the others.

Reply:
Looks like Kelvin will fix https://github.com/ethereum-optimism/contracts/issues/243#issue-814828116 with #300, so this can PR of mine can be closed. 

At the time, I didn't exactly know how to address #243, so this was my best shot 😅.